lib/plugins/aws/invokeLocal/index.js

```diff
@@ -266,10 +266,16 @@ class AwsInvokeLocal {
     if (['java8', 'java11'].includes(runtime)) {
       const className = handler.split('::')[0];
       const handlerName = handler.split('::')[1] || 'handleRequest';
-      const artifact =
-        this.options.package && this.options.package.artifact
-          ? this.options.package.artifact
-          : this.serverless.service.package.artifact;
+      const artifact;
+      if (this.serverless.service.package.individually === true) {
+        artifact = this.options.functionObj.package && this.options.functionObj.package.artifact ?
+          this.options.functionObj.package.artifact :
+          undefined
+      }
+      else {
+        artifact = this.serverless.service.package.artifact
```

Comment:
This changes how we resolve artifact for non `package.individually: true` case. Is that intended?